README.md

```diff
@@ -1237,10 +1237,101 @@ class MyClass
 
 ### PhpDoc on methods
 
-We put PhpDoc comment on all methods with exception of constructors (can be skipped in some cases).
+#### No PhpDoc on fully strictly typed methods
 
-We put PhpDoc comment on constructors when IDE (for example PhpStorm) cannot guess classes of attributes,
-return type etc. It’s optional otherwise. For example, if we inject some scalar type, we must put PhpDoc comment.
+We put a PhpDoc comment on methods where the return and argument types cannot be determined by the IDE. For example, 
```

Comment:
`cannot be determined by the IDE` - this is a bit confusing, as IDE can guess return type from `return new Money();`, even if we don't have `: Money` defined.
I think it would be clearer to call this `is not in the definition` or some other way.